### PR TITLE
MKS-49 Adjust routing

### PR DESCRIPTION
**Pull request by BohdanHamurda**

### CODE REVIEW

File: Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/Controllers/StudentController.cs [hunk 1]
```diff
@@ -27,31 +28,109 @@ public IActionResult Get()
...
+                return StatusCode(StatusCodes.Status500InternalServerError);
+            }
         }
 
 
         [HttpPost]
         public IActionResult Post([FromBody] Students value)
         {
-            return null;
+            try
```
Comment (bvzayats) [inline Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/Controllers/StudentController.cs]: BL of Post and Put endpoints should be implemented in Additional task

Comment (BohdanHamurda) [reply Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/Controllers/StudentController.cs]: Done

File: Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/Controllers/StudentController.cs [hunk 2]
```diff
@@ -41,17 +59,37 @@ public IActionResult Post([FromBody] Students value)
...
-        [HttpPut]
+        [HttpPut("{id}")]
         public IActionResult Put(int id, [FromBody] Students value)
         {
             return null;
         }
 
 
-        [HttpDelete]
+        [HttpDelete("{id}")]
```
Comment (bvzayats) [inline Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/Controllers/StudentController.cs:69]: I can make mistakes. But you have to read a Task description! Delete also should be implemented in another task.

Comment (BohdanHamurda) [reply Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/Controllers/StudentController.cs:69]: Done

**Review by Magfiozy — APPROVED** [review]

**Review by bvzayats — APPROVED** [review]

**Pull request merged**